nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/osgi/EmbeddedOSGiGlassFishRuntime.java

```diff
@@ -83,6 +84,9 @@ public EmbeddedOSGiGlassFishRuntime(BundleContext context) {
     @Override
     public synchronized GlassFish newGlassFish(GlassFishProperties gfProps) throws GlassFishException {
         try {
+            for (Object key : gfProps.getProperties().keySet()) {
+                System.out.println(key + ": " + gfProps.getProperties().get(key));
```

Comment:
Can we use Logger instead of System.out.println?

Reply:
removed